### ISSUE TITLE
Distal radius fracture malunion

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10749]
**Preferred term label:**
Distal radius fracture malunion

**Synonyms**

**Definition (free text, please give PubMed ID)**
AP or lateral wrist radiograph showing a healed distal radius fracture in a nonanatomic position

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0045008

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Distal radius malunion

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
In order to avoid a profusion of bone x fracture x fracture type terms, we introduced a subhierachy of fracture types including
[Displaced fracture HP:4000053](https://hpo.jax.org/browse/term/HP:4000053)
Would this be able to capture it? Is displaced actually synonymous with malunion in this context?

**Comment by MickeySegal** [comment]
Displaced is a typical assessment for an acute fracture.  The intention here is to describe a chronic displacement "showing a healed distal radius fracture in a nonanatomic position".

**Comment by pnrobinson** [comment]
@MickeySegal
Sorry for the late response to this.
My question was whether we should add a Modifer that can be applied to any fracture.
[Displaced fracture HP:4000053](https://hpo.jax.org/browse/term/HP:4000053) is one such modifier.
Fracture malunion Could be another such modifier with definition:

Malunion of a fracture occurs when a fracture heals in a non-anatomical position, which can result in functional impairment.

**Comment by MickeySegal** [comment]
If using this approach, fracture malunion would be the appropriate choice.
